src/store/filestore.rs

```diff
@@ -49,206 +43,196 @@ impl Store for LocalFileStore {
     }
 
     fn delete_annotation<T>(&self, name: &str, version: &str) -> Result<()> {
-        // Search the name ver index for the hash
-        let hash = self.get_hash_from_name_ver_tree::<T>(name, version)?;
+        let hash = self.lookup_hash::<T>(name, version)?;
+        let (count, _) = Self::parse_annotation_path(
+            &self.make_path::<T>(&hash, Self::make_annotation_relpath("*", "*")),
+        )?
+        .size_hint();
+        if count == 1 {
+            return Err(OrcaError::from(Kind::DeletingLastAnnotation(
```

Comment:
Same thing as above about the point relating to annoationless models

Reply:
This one is related to requiring annotation on `save_model` and consequently, preventing the last annotation from being deleted so it won't *disappear* from `list_model` output. See my other [comment](https://github.com/walkerlab/orcapod/pull/42#discussion_r1827652408) for context.